orchestra/communication/staffing.py

```diff
@@ -217,7 +316,7 @@ def get_available_requests(worker):
         StaffingRequestInquiry.objects
         .filter(request__in=remaining_requests)
         .filter(communication_preference__worker=worker)
-        .order_by('request__task__start_datetime'))
+        .order_by('-request__task__project__priority', 'request__created_at'))
```

Comment:
does ordering by priority first ever leave us in a place where we don't staff an old (e.g.) enhancement because we're always getting new launch tasks?

Reply:
we can always update a project's priority in VIP if it gets starved out.